pkg/kubelet/winstats/cpu_topology.go

```diff
@@ -233,7 +234,7 @@ func convertWinApiToCadvisorApi(buffer []byte) (int, int, []cadvisorapi.Node, er
 			}
 
 		default:
-			klog.V(4).Infof("Not using Windows CPU relationship type: %d", info.Relationship)
```

Comment:
It's interesting that this non-structured logging was here. Theoretically it should raise lint errors as all pkg/kubelet [is supposed to use structured logging](https://github.com/kubernetes/kubernetes/blob/master/hack/logcheck.conf#L18).

Reply:
We have a gap with linting: it doesn't run for Windows-only code like this one here. Might be worth closing?

cc @BenTheElder @aojea

Reply:
We also don't for e.g. GOOS=arm64, right?

We have some very specific linting for "does it typecheck" that targets all platforms, with a custom tool.

I'd be a bit concerned about the cost to golangci-lint for all platforms, but I suppose we're more likely to have build tagged files for linux,windows generally speaking.

Reply:
It's definitely gradual and a question of "bang for the buck". My expectation is that we have more Windows-only code than arm64-only code.

Reply:
I had a brief look by setting GOOS=windows in the script. There are several valid reports where Windows code is not following conventions that are enforced when building for Linux. I wonder whether SIG Windows is aware.

Anyway, I'm not going to pursue this further. If I did, I would have to fix all those issues...